### PR TITLE
Duplicates merged

### PR DESCRIPTION
**Pull request by loma-one**
Duplicates merged & emoji enabled/disabled

### CODE REVIEW

File: unicode_smilies/unicode_smilies.php [hunk 1]
```diff
@@ -549,19 +351,24 @@ function unicode_smilies_smilies(&$a,&$b) {
 //	Smilies::add($b, ':deaf man:', '&#x1F9CF&#x200D&#x2642&#xFE0F');
 //	Smilies::add($b, ':deaf woman:', '&#x1F9CF&#x200D&#x2640&#xFE0F');
 	Smilies::add($b, ':person bowing:', '&#x1F647');
+	Smilies::add($b, ':bow:', '&#x1F647;');
 	Smilies::add($b, ':man bowing:', '&#x1F647&#x200D&#x2642&#xFE0F');
 	Smilies::add($b, ':woman bowing:', '&#x1F647&#x200D&#x2640&#xFE0F');
 	Smilies::add($b, ':person facepalming:', '&#x1F926');
+    Smilies::add($b, ':facepalm:', '&#x1F926;');
```
Comment (MrPetovan) [inline unicode_smilies/unicode_smilies.php]: Code standards: Please normalize the indentation.
```suggestion
	Smilies::add($b, ':facepalm:', '&#x1F926;');
```

File: unicode_smilies/unicode_smilies.php [hunk 2]
```diff
@@ -685,13 +499,13 @@ function unicode_smilies_smilies(&$a,&$b) {
 	Smilies::add($b, ':baby chick:', '&#x1F424');
 	Smilies::add($b, ':front-facing baby chick:', '&#x1F425');
 	Smilies::add($b, ':bird:', '&#x1F426');
-	Smilies::add($b, ':penguin:', '&#x1F427');
+	Smilies::add($b, ':tux:', '&#x1F427');
```
Comment (MrPetovan) [inline unicode_smilies/unicode_smilies.php:502]: Why this change?

File: unicode_smilies/unicode_smilies.php [hunk 3]
```diff
@@ -862,6 +679,7 @@ function unicode_smilies_smilies(&$a,&$b) {
 	Smilies::add($b, ':cookie:', '&#x1F36A');
 	Smilies::add($b, ':birthday cake:', '&#x1F382');
 	Smilies::add($b, ':shortcake:', '&#x1F370');
+    Smilies::add($b, ':cake:', '&#x1F370;');
```
Comment (MrPetovan) [inline unicode_smilies/unicode_smilies.php]: Code standards: Please normalize indentation.
```suggestion
	Smilies::add($b, ':cake:', '&#x1F370;');
```

File: unicode_smilies/unicode_smilies.php [hunk 4]
```diff
@@ -1191,11 +1015,20 @@ function unicode_smilies_smilies(&$a,&$b) {
 	Smilies::add($b, ':skis:', '&#x1F3BF');
 	Smilies::add($b, ':sled:', '&#x1F6F7');
 	Smilies::add($b, ':curling stone:', '&#x1F94C');
+    Smilies::add($b, ':cycling:', '&#x1F6B4;');
```
Comment (MrPetovan) [inline unicode_smilies/unicode_smilies.php]: Code standards: Please normalize indentation.
```suggestion
	Smilies::add($b, ':cycling:', '&#x1F6B4;');
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: unicode_smilies/unicode_smilies.php (hunk 2 above)
Comment (annando) [reply unicode_smilies/unicode_smilies.php:502]: Question is, if there is still an entry with "penguin" in that list.

Comment (loma-one) [reply unicode_smilies/unicode_smilies.php:502]: The duplicate from version 1.0 was written with Tux. I have adopted the name because it is probably often used out of habit. http://www.unicode.org/emoji/charts/full-emoji-list.html provides for penguin.

**Review by MrPetovan — APPROVED** [review]
